lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -139,7 +139,7 @@ def update(frame):
         else:
             # print(signal.getsignal(signal.SIGINT))
```

Comment:
Hmmm... I left a print comment in here... bad form!

Reply:
Removed this comment

Reply:
And it was mine, btw =)